### PR TITLE
initial push of code from demo site

### PR DESCRIPTION
**Pull request by rhagarty**
I pulled everything that I thought was relevant from the original demo zip file that was sent to me.

I seeded the README with some DSX/HDP info from the feature-engineering repo. Architecture diagram + flow steps obviously need to be updated.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Mac OS
+.DS_Store
+
+# all dataset files except original wine data
+datasets
+!datasets/Wine.csv
```
Comment (markstur) [inline .gitignore:6]: does it make sense to move this file into data/input or something to simplify this and maybe be more like our usual dirs

File: README.md [hunk 2]
```diff
@@ -1 +1,120 @@
...
+
+> **What is IBM DSX Local?** DSX Local is an on premises solution for data scientists and data engineers. It offers a suite of data science tools that integrate with RStudio, Spark, Jupyter, and Zeppelin notebook technologies. And yes, it can be configured to use HDP, too.
+
+> **What is the IBM Deployment Manager?** The Deployment Manager is a DSX Local tool that provides users the ability to create and train machine learning models. Users can also deploy their models to make them available to a wider audience.
+
+This repo contains 
+
+When you have completed this code pattern, you will understand how to:
+
+* Load data into Spark DataFrames and use Spark's machine learning library (MLlib) to train a
```
Comment (markstur) [inline README.md:19]: sill WIP, huh?

**Review by markstur — APPROVED** [review]
ok for W-I-P getting started

File: .gitignore (hunk 1 above)
Comment (rhagarty) [reply .gitignore:6]: good call - I will change this

File: README.md (hunk 2 above)
Comment (rhagarty) [reply README.md:19]: yeah, wanted to leave the train of thought, but the rest of the line needs to be updated to the specify the techniques to be used. So leave as is for now.

**Pull request merged**